generators/theme/generate.php

```diff
@@ -21,9 +25,20 @@ function generate() {
       mkdir($this->target_dir);
     }
 
-    $this->recurse_copy(dirname(__FILE__) . "/template", $this->target_dir);
-
+    (new Git($this->target_dir))->clone_repo($this->whippet_theme_repo);
     // Delete the spurious .git file
-    unlink("{$this->target_dir}/.git");
+    chdir("{$this->target_dir}");
+    exec('rm -rf .git');
```

Comment:
We don't need to resort to exec() here. We have a function to do this: https://github.com/dxw/whippet/blob/1059e5ccd50477f5d2692cf794b54df3d1d30975/src/Modules/Helpers/WhippetHelpers.php#L114-L131